### ISSUE TITLE
Honor .formatter.exs line_length in editor.ruler

### ISSUE THREAD

**Issue by TheFirstAvenger** [elixir-lsp/vscode-elixir-ls#395]
Is it possible to read the .formatter.exs 'line_length` property and use that to override the `edtior.ruler` value in VSCode for that project, making the vertical line length indicator always accurate to the current project? If it is not present in that file, then the formatter default of 98 should be applied.

**Comment by lukaszsamson** [comment]
Interesting idea. This would require evaluating `Mix.Tasks.Format.formatter_for_file` for each file but we can add a custom LSP command doing that. Do you know which vscode API can override `edtior.rulers`?

**Comment by TheFirstAvenger** [comment]
Hmmm... upon further digging, it looks like there  is [an open PR](https://github.com/microsoft/vscode/pull/124879) for VSCode to support this.

**Comment by TheFirstAvenger** [comment]
Yeah that makes sense. I will keep an eye on those PRs and if they happen to merge I will revisit this.
